torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -23,6 +24,8 @@ const std::string kNumIdleThreads = "agent.num_idle_threads";
 const std::string kClientActiveCalls = "agent.client_active_calls";
 const std::string kServerActiveCalls = "agent.server_active_calls";
 const std::string kServerActiveAsyncCalls = "agent.server_active_async_calls";
+const std::string kTensorPipeRPCTimeoutErrorStr =
```

Comment:
nit: kTensorPipeRpcTimeoutErrorStr

Reply:
nit: no need for the `TensorPipe` prefix, this is in anonymous namespace, it's local to this file